### DISCUSSION TITLE
How to safely update to the latest version

### DISCUSSION THREAD

**Discussion by sillywan** [supermerill/SuperSlicer#1384] [Q&A]
I have been useing SS for a couple of months now and I have updated the software once allready. It is a confusing proeedure as it is and I am not sure I got it done right the last time. I think I have some stuff hanging around from the original install. I am wanting to install the latest version 2.3.56.6. I am not sure how to go about this. I have put the files in program files in a directory called superslicer. I then made shortcuts to them. I must have done ok as it is working great. How do I install the new version? Do I have to delete the old version first? It is not installed thru windows so there is no uninstall. Its installed almost like a dos file. Unzipped into a directory, and that directory put into program files and shortcuts made to the executables. How do I keep my profiles and machine information? I love this slicer, I don't even use cura anymore. Can someone help me safely get this done? I have not had any issues with the earlier version so far but there have been some major bug fixes so it is probably a worthy update. Thank you for this awesome piece of software, and your help

**Comment by supermerill** [comment]
it's not mandatory to delete the old directory, 
 * you can replace everything.
 * or you can delete the folder and unzip the new one

> How do I keep my profiles and machine information?

It's saved into your `appdata/roaming/superslicer` setting directory, so you don't have anything to do (unless you used the2.3.55, as these ones used the `superslicer-alpha` setting directory). You can check it by using `Help -> Show Configuration Folder`
You can backup your setting directory if you want to be extra careful.

**Reply by sillywan** [reply]
Thank you so much for the information. I am a real fan of this slicer. It is running so well I do not want to mess it up by not knowing. So I asked first. Thank you for your quick response, and the news I wanted to hear ;-) Have a wonderfull day.
